mbed_app.json

```diff
@@ -2,7 +2,11 @@
     "config": {
         "regression-test": 1,
         "psa-compliance-test": 0,
-        "wait-for-sync": 1
+        "wait-for-sync": 1,
+        "psa-test-ram": {
+            "help": "Start of the memory used by the PSA test framework, see api-tests/platform/targets/<target>/target.cfg in psa-arch-tests",
+            "value": "0x20060000"
```

Comment:
Is this consistent for all the targets? Is the CI testing B1 and S1?

Reply:
This is consistent for B1 and S1. The CI tests only S1, but we need this PR for `mbedgt` on desk too.